neofetch

```diff
@@ -1127,6 +1133,24 @@ get_model() {
             model="$(/usr/bin/uname -M)"
         ;;
 
+        "HP-UX")
+            machine_type="$(model | awk -F'/' '{print $1}')"
```

Comment:
What's the output of `model`? We could read it into an array and save calling it (and awk) multiple times.

Also, change all occurrences of `[ ]` to `[[ ]]`.

Reply:
`model` output on PA-RISC systems seems to most often be a three or two-part string delimiting the system's model using the old HP 9000 naming convention, on older systems the second and third component correspond to the model and submodel (e.g. `9000/715/100`) while on newer systems it seems to correspond more to family and model (`9000/785/C3000`) hence why I'm cutting up the string differently depending on whether characters are present in the last string or not.

It's definitely consistent enough that you could just parse it into an array, I'll take a look at it.

Reply:
On IA64 systems, the usual format would be `ia64 hp server <actual model>`.

Reply:
I think I found some useful resources with `model` strings from a number of systems:
ftp://ftp.mrynet.com/operatingsystems/HP-MPE/docs.hp.com/en/5991-6552/ch02s03.html
ftp://ftp.mrynet.com/operatingsystems/HP-MPE/docs.hp.com/en/5991-1247B_ed2/ch02s04.html
There's even some Superdome output on here, which is great. There's also a couple four-part strings by the looks of it.